docs/admin-get-started.md

```diff
@@ -34,7 +34,9 @@ If you have enabled authentication, you need to provide an auth configuration to
 </TabItem>
 <TabItem value="REST API">
 
-You can find details for the REST API exposed by Pulsar brokers in this {@inject: rest:document:/}.
+You can find details for the REST API exposed by Pulsar brokers in the [REST API doc](reference-rest-api-overview.md).
+
+If you want to test REST APIs in postman, you can use the REST API JSON files [here](https://pulsar.apache.org/swagger/master/).
```

Comment:
@yuweisung I add this instruction here **temporarily** because:

1) From the user learning perspective, users will be interested in "testing" it after "trying" it

2) From the content perspective, for existing content, here is the most relevant part. 

We can relocate this instruction to other places if we have more related content in the future.